stubs/tensorflow/tensorflow/python/trackable/ressource.pyi

```diff
@@ -0,0 +1,7 @@
+from tensorflow.python.trackable.base import Trackable
+
+class _ResourceMetaclass(type): ...
+
+# Internal type that is commonly used as a base class
+# and some public apis the signature needs it.
+class CapturableResource(Trackable, metaclass=_ResourceMetaclass): ...
```

Comment:
Not a full review:

There is no file `ressource.py` in the source. Did you mean to name this file `resource.pyi`? Also, `resource.py` contains more items that the stubs. If you don't want to add them add this time, please add the following to the stubs:

```python
from _typeshed import Incomplete

def __getitem__(name: str) -> Incomplete: ...  # incomplete
```

Reply:
Fixed in bd198e492, thanks!